test/plugins/jobs.test.js

```diff
@@ -330,20 +330,20 @@ describe('job plugin test', () => {
                 assert.equal(reply.statusCode, 200);
                 assert.calledWith(job.getBuilds, {
                     sort: 'descending',
-                    sortBy: 'createTime'
+                    sortBy: 'createTime',
+                    readOnly: true
                 });
                 assert.deepEqual(reply.result, testBuilds);
             }));
 
         it('returns 200 for getting builds with query params', () => {
-            options.url = `/jobs/${id}/builds?fetchSteps=false&readOnly=true`;
+            options.url = `/jobs/${id}/builds?fetchSteps=false&readOnly=false`;
 
             return server.inject(options).then(reply => {
                 assert.equal(reply.statusCode, 200);
                 assert.calledWith(job.getBuilds, {
                     sort: 'descending',
-                    sortBy: 'createTime',
-                    readOnly: true
```

Comment:
Is there any reason not to explicitly state readOnly:false here?

Reply:
It is because `getBuilds` is called that way.
https://github.com/screwdriver-cd/screwdriver/blob/2049371e4a22adb0aa82a172c3f604914ea907b6/plugins/jobs/listBuilds.js#L33-L35

Reply:
Thank you. I understood it.